### PR TITLE
Use scalajs-react-table for MagnitudesForm.

### PR DESCRIPTION
**Pull request by toddburnside**
[ch568]
Magnitudes table should look exactly the same as the previous version with the SemanticUI table.

The code for adding controls to columns is MUCH simpler than for ag-grid (ReactTableHelpers).

image

### CODE REVIEW

**Comment by cquiroz** [comment]
Do you have a screenshot?

File: common/src/main/scala/explore/utils/ReactTableHelpers.scala [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import monocle.Lens
+import react.semanticui.elements.button.Button
+import reactST.reactTable.mod._
+
+import scalajs.js
+
+object ReactTableHelpers {
+
+  /**
+   * Create a AgGridColumn Builder for a column which edits the value in a FormInputEv.
```
Comment (cquiroz) [inline common/src/main/scala/explore/utils/ReactTableHelpers.scala:24]: `AgGridColumn`?

File: common/src/main/scala/explore/utils/ReactTableHelpers.scala [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        FormInputEV(id = newId,
+                    value = va.zoom(lens),
+                    validFormat = validFormat,
+                    changeAuditor = changeAuditor,
+                    disabled = disabled,
+                    modifiers = modifiers
+        )
+      }
+      .build
+      .cmapCtorProps[(CellProps[View[A], _]) with js.Object](_.cell.row.original)
```
Comment (cquiroz) [inline common/src/main/scala/explore/utils/ReactTableHelpers.scala:53]: `neat`

**Review by cquiroz — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/utils/ReactTableHelpers.scala (hunk 1 above)
Comment (toddburnside) [reply common/src/main/scala/explore/utils/ReactTableHelpers.scala:24]: Oops.

**Comment by toddburnside** [comment]
> Do you have a screenshot?

I can post one, but it looks exactly the same as what is in staging/production.

File: common/src/main/scala/explore/utils/ReactTableHelpers.scala (hunk 1 above)
Comment (toddburnside) [reply common/src/main/scala/explore/utils/ReactTableHelpers.scala:24]: Looks like I did too much copy/paste of documentation from the ag-grid implementation. I'll go through that.

**Pull request merged**